### DISCUSSION TITLE
OPS, Ops, or ops?

### DISCUSSION THREAD

**Discussion by JonathanMcCormickJr** [nanovms/ops#1346] [Q&A] [answered]
As I am proofreading the documentation, I notice that the "OPS" name currently has many variants of capitalization. Is there a preferred way to write it **outside** of coding?

# OPS
* Gives impression that it might be an acronym. 🙃
  * One-Process System?
  * Our People are Smart?
  * Only Perform Single process?

# Ops
* Standard proper noun capitalization.

# `ops`
* Good for coding.

**Comment by eyberg** [comment, marked as answer]
no strong opinions but typically 'ops' for shell, documentation purposes (cause it typically is documenting said shell usage) and 'OPS' for other purposes (heavily used in various graphical assets), 'ops' can be used in chat; 'Ops' might be found in things like articles
